roles/velero/defaults/main.yml

```diff
@@ -22,29 +22,32 @@ velero_upstream_tag: v1.5.2
+velero_openshift_plugin_image: "{{ registry }}/{{ project }}/{{ velero_openshift_plugin_repo }}"
 velero_openshift_plugin_repo: "{{ lookup( 'env', 'VELERO_OPENSHIFT_PLUGIN_REPO') }}"
 velero_openshift_plugin_version: "{{ lookup( 'env', 'VELERO_OPENSHIFT_PLUGIN_TAG') }}"
 velero_csi_plugin_image: "{{ upstream_registry }}/{{ velero_upstream_project }}/{{ velero_csi_plugin_repo }}"
 velero_csi_plugin_repo: "{{ lookup('env', 'VELERO_CSI_PLUGIN_REPO') }}"
 velero_csi_plugin_version: "{{ lookup('env', 'VELERO_CSI_PLUGIN_TAG') }}"
-velero_vsphere_plugin_image: "{{ upstream_registry }}/{{ velero_vsphere_upstream_project }}/{{ velero_vsphere_plugin_repo }}"
+velero_vsphere_plugin_image: "{{ upstream_registry }}/{{ velero_vsphere_upstream_project }}/{{ velero_vsphere_plugin_repo }}" # TODO: Remove this. Vsphere won't be supported
 velero_vsphere_plugin_repo: "{{ lookup('env', 'VELERO_VSPHERE_PLUGIN_REPO') }}"
 velero_vsphere_plugin_version: "{{ lookup('env', 'VELERO_VSPHERE_PLUGIN_TAG') }}"
 velero_restic_restore_helper_image: "{{ registry }}/{{ project }}/{{ velero_restic_restore_helper_repo }}"
 velero_restic_restore_helper_repo: "{{ lookup( 'env', 'VELERO_RESTIC_RESTORE_HELPER_REPO') }}"
-velero_restic_restore_helper_version: "{{ velero_upstream_tag if use_upstream_images is defined and use_upstream_images|bool == true else lookup( 'env', 'VELERO_RESTIC_RESTORE_HELPER_TAG') }}"
+velero_restic_restore_helper_version: "{{ lookup( 'env', 'VELERO_RESTIC_RESTORE_HELPER_TAG') }}"
 velero_aws_plugin_image: "{{ registry }}/{{ project }}/{{ velero_aws_plugin_repo }}"
 velero_aws_plugin_repo: "{{ lookup( 'env', 'VELERO_AWS_PLUGIN_REPO') }}"
-velero_aws_plugin_version: "{{ velero_upstream_plugin_tag if use_upstream_images is defined and use_upstream_images|bool == true else lookup( 'env', 'VELERO_AWS_PLUGIN_TAG') }}"
+velero_aws_plugin_version: "{{ lookup( 'env', 'VELERO_AWS_PLUGIN_TAG') }}"
 velero_gcp_plugin_image: "{{ registry }}/{{ project }}/{{ velero_gcp_plugin_repo }}"
 velero_gcp_plugin_repo: "{{ lookup( 'env', 'VELERO_GCP_PLUGIN_REPO') }}"
-velero_gcp_plugin_version: "{{ velero_upstream_plugin_tag if use_upstream_images is defined and use_upstream_images|bool == true else lookup( 'env', 'VELERO_GCP_PLUGIN_TAG') }}"
+velero_gcp_plugin_version: "{{ lookup( 'env', 'VELERO_GCP_PLUGIN_TAG') }}"
 velero_azure_plugin_image: "{{ registry }}/{{ project }}/{{ velero_azure_plugin_repo }}"
 velero_azure_plugin_repo: "{{ lookup( 'env', 'VELERO_AZURE_PLUGIN_REPO') }}"
-velero_azure_plugin_version: "{{ velero_upstream_plugin_tag if use_upstream_images is defined and use_upstream_images|bool == true else lookup( 'env', 'VELERO_AZURE_PLUGIN_TAG') }}"
+velero_azure_plugin_version: "{{ lookup( 'env', 'VELERO_AZURE_PLUGIN_TAG') }}"
+velero_registry_tag: "{{lookup( 'env', 'VELERO_REGISTRY_IMAGE_TAG') }}"
+velero_registry_repo: "{{lookup( 'env', 'VELERO_REGISTRY_IMAGE_REPO') }}"
```

Comment:
Same question here as well. `VELERO_REGISTRY_REPO` ? As you are setting this as environment variable ?